mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/mapping/impl/BridgeResolver.java

```diff
@@ -325,6 +329,20 @@ private void addDefaults() {
 					.valueBinder( new StaticValueBinder<>( GeoPoint.class, DefaultGeoPointBridge.INSTANCE ) );
 			exactType( GeoPoint.class )
 					.identifierBridge( DefaultGeoPointBridge.INSTANCE );
+
+			// arrays for vector fields:
+			exactType( byte[].class )
+					.valueBridge( DefaultBytePrimitiveArrayBridge.INSTANCE )
+					.identifierBridge( DefaultBytePrimitiveArrayBridge.INSTANCE );
+			exactType( float[].class )
+					.valueBridge( DefaultFloatPrimitiveArrayBridge.INSTANCE )
+					.identifierBridge( DefaultFloatPrimitiveArrayBridge.INSTANCE );
+			exactType( Byte[].class )
+					.valueBridge( DefaultByteArrayBridge.INSTANCE )
+					.identifierBridge( DefaultByteArrayBridge.INSTANCE );
+			exactType( Float[].class )
+					.valueBridge( DefaultFloatArrayBridge.INSTANCE )
+					.identifierBridge( DefaultFloatArrayBridge.INSTANCE );
```

Comment:
I'm not sure this makes a lot sense... But if you think it does, okay.

Note however that `List<Byte>` and `List<Float>` would make a lot more sense to support. Maybe create a follow-up Jira for that?